compile-release-tools

```diff
@@ -59,9 +33,10 @@ compile() {
 }
 
 main() {
+  cd "$(dirname "${BASH_SOURCE[0]}")" \
+    && cd "$( git rev-parse --show-toplevel )"
```

Comment:
I really like this pattern, however I will note that it does require this script to be used from a git clone... I think that's reasonable here but I'm on the fence about always using this or not. 

For repos we expect source tarballs to be useful for it's not great.

Reply:
I don't have a strong opinion on that. Happy to switch to `cd "$(dirname "${BASH_SOURCE[0]}")"` only and rely on the fact, that this script will probably never be moved around in the repo.